app/views/events/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+
+<div>
+
+<h1>Past Events</h1>
+
+<table>
+<tr>
+    <th>Description</th>
+    <th>Show</th>
+</tr>
+<%@events.past.each do |event|%>
+<tr>
+
+<td><%=event.description%></td>
+<td><%= link_to "watch event", "/show_event?id=#{event.id}" %></td>
+
+</tr>
+<%end%>
+</table>
+
+</div>
+
+<div>
+<h1>Upcoming Events</h1>
+
+<table>
+<tr>
+    <th>Description</th>
+    <th>Show</th>
+    <th>Attend Event</th>
+</tr>
+<%@events.upcoming.each do |event|%>
+<tr>
+
+<td><%=event.description%></td>
+<td><%= link_to "watch event", "/show_event?id=#{event.id}" %></td>
+<% if Attendance.where(user_id: current_user.id, event_id: event.id)==[]%>
+<td><%= button_to 'attend event',  "/attendance?id=#{event.id}"%><td>
```

Comment:
I'm not able to attend an event because all events are classified as past events...

Reply:
we fix that bug you can attend all future events